.travis.yml

```diff
@@ -20,14 +20,21 @@ env:
   matrix:
     - INSTANCE: default-debian-9-2019-2-py3
     - INSTANCE: default-ubuntu-1804-2019-2-py3
-    - INSTANCE: default-centos-7-2019-2-py2
-    - INSTANCE: default-fedora-29-2019-2-py2
-    - INSTANCE: default-opensuse-423-2018-3-py2
-    - INSTANCE: default-debian-8-2018-3-py2
+    - INSTANCE: default-centos-7-2019-2-py3
+    - INSTANCE: default-fedora-29-2019-2-py3
+    - INSTANCE: default-opensuse-leap-15-2019-2-py3
+    - INSTANCE: default-debian-9-2018-3-py2
     - INSTANCE: default-ubuntu-1604-2018-3-py2
-    - INSTANCE: default-fedora-28-2018-3-py2
+    - INSTANCE: default-centos-7-2018-3-py2
+    - INSTANCE: default-fedora-29-2018-3-py2
+    # TODO: Use this when fixed instead of `opensuse-leap-42`
+    # - INSTANCE: default-opensuse-leap-15-2018-3-py2
```

Comment:
This can then be enabled, or should we remove it completely from here?

Reply:
Thanks @javierbertoli!  I've just re-enabled the failing image.  Here are some refs for you, actually had two failures here:

* `opensuse-leap-15`: https://github.com/netmanagers/salt-image-builder/issues/2
* `opensuse-leap-42`: https://github.com/netmanagers/salt-image-builder/issues/3

Now to undo that again...

Reply:
Actually, that second failure was only for the `postgres-formula`, not the `template-formula`.

Reply:
Mind you add a link to the issue in the case of [opensuse-leap-15](https://github.com/netmanagers/salt-image-builder/issues/2) so we don't lose track of the reasons we disable things?